src/adv.py

```diff
@@ -33,19 +65,269 @@
 room['narrow'].n_to = room['treasure']
 room['treasure'].s_to = room['narrow']
 
+
 #
 # Main
 #
 
-# Make a new player object that is currently in the 'outside' room.
+my_player = Player('', room['outside'])
 
-# Write a loop that:
-#
-# * Prints the current room name
-# * Prints the current description (the textwrap module might be useful here).
-# * Waits for user input and decides what to do.
-#
-# If the user enters a cardinal direction, attempt to move to the room there.
-# Print an error message if the movement isn't allowed.
-#
-# If the user enters "q", quit the game.
+
+def main():
+    # Make a new player object that is currently in the 'outside' room.
```

Comment:
consider updating/moving this comment. os.system('clear') could use its own comment (why are you doing that?)